Before: 8c6e5ef21ac9c40912aba7ba2fe7824aeaa3707024e0495fed9005ad55ede9f1
After: 4bbf052716898c468432610b84777d66907d5f48ddfa1652d87305f773efe334
Patch: b1261653ac43e47439f1cc7acda519c3b3207c15804b04592a2b0dfc99b4af1b
--- Gemfile.corvid	2012-07-22 20:44:49.332250333 +1000
+++ Gemfile.corvid	2012-07-22 21:28:12.010137438 +1000
@@ -10,7 +10,8 @@
 
 group :test do
   gem 'ci_reporter', require: false
-  gem 'simplecov', '>= 0.6.4', require: false
+  #gem 'simplecov', require: false
+  gem 'simplecov', require: false, git: 'git://github.com/japgolly/simplecov.git', branch: 'nan_fix'
 end if Dir.exists?('test')
 
 group :test_unit do
